### PR TITLE
Retry if can't find instance right away

### PR DESCRIPTION
**Pull request by tlunter**
@bfulton @adamjt @nahiluhmot 

When an instance can't be found via EC2's API, but we found it with the ASG API, retry since it'll be there soon.

### CODE REVIEW

File: lib/aerosol/instance.rb [hunk 1]
```diff
@@ -44,8 +46,17 @@ def self.request_all
...
   def describe!
     ensure_present! :id
+    attempts ||= 0
     result = Aerosol::AWS.compute.describe_instances('instance-id' => id).body
     result['reservationSet'].first['instancesSet'].first rescue nil
+  rescue Fog::Compute::AWS::NotFound => ex
+    if attempts < 3
+      attempts += 1
+      logger.error "#{ex.class}: #{ex.message} - retrying"
+      retry
```
Comment (adamjt) [inline lib/aerosol/instance.rb]: Should there to be a sleep period in here between retries?

Comment (tlunter) [inline lib/aerosol/instance.rb]: That's probably a good idea.  Not sure how long of a wait it should be.  Just 1 second?

Comment (adamjt) [inline lib/aerosol/instance.rb]: Sounds good to me!

File: lib/aerosol/instance.rb [hunk 2]
```diff
@@ -44,8 +46,17 @@ def self.request_all
...
+    else
+      raise
```
Comment (bfulton) [inline lib/aerosol/instance.rb]: Will this raise the original exception?

File: lib/aerosol/instance.rb (hunk 1 above)
Comment (bfulton) [inline lib/aerosol/instance.rb]: How about 6 retries with 5s sleep, so we give it 30s to register?

File: lib/aerosol/instance.rb (hunk 2 above)
Comment (tlunter) [inline lib/aerosol/instance.rb]: Yes

File: lib/aerosol/instance.rb (hunk 1 above)
Comment (tlunter) [inline lib/aerosol/instance.rb]: Seems reasonable to me.

**Pull request closed without merging**